partner_multi_relation/models/res_partner_relation_type.py

```diff
@@ -83,6 +83,29 @@ def get_partner_types(self):
             ('p', _('Person')),
         ]
 
+    @api.model
+    def _end_active_relations(self, relations):
+        """End the relations that are active.
+
+        If a relation has a start date in the future, it will be deleted.
+
+        If a relation has a start date in the future and no end date OR
```

Comment:
These three lines about current relations seem to me not wholly correct. Before these lines is the line that explains future relations will be deleted.

So why not simply say:
```If a relation is current, that is, if it has a start date in the past and end date in the future (or no end date), the end date will be set to the current date.```

Reply:
@NL66278 ok fixed.